lib/run_loop/life_cycle/core_simulator.rb

```diff
@@ -453,11 +453,19 @@ def wait_for_device_state(target_state)
       end
 
       # @!visibility private
+      def csstore_file
+        @csstore_file ||= File.join(device_data_dir, 'Library', 'Caches', 'com.apple.LaunchServices-134.csstore')
```

Comment:
@sapieneptus 
This is great - enough to close this issue - but can you make a new run-loop issue to replace this implementation with one that matches with regex?

We think that name of this file will change per Xcode version.